apps/journeys/pages/index.tsx

```diff
@@ -8,24 +8,27 @@ import {
   GetJourneys,
   GetJourneys_journeys as Journey
 } from '../__generated__/GetJourneys'
+import { themes } from '@core/shared/ui'
 
 interface JourneysPageProps {
   journeys: Journey[]
 }
 
 function JourneysPage({ journeys }: JourneysPageProps): ReactElement {
   return (
-    <Container>
-      {journeys.map(({ id, title }) => (
-        <Box key={id} my={2}>
-          <Link href={`/${id}`} passHref>
-            <Button variant="contained" color="primary" fullWidth>
-              {title}
-            </Button>
-          </Link>
-        </Box>
-      ))}
-    </Container>
+    <ThemeProvider theme={themes.light}>
```

Comment:
this should use JourneyThemeProvider
```suggestion
    <JourneyThemeProvider themeName={ThemeName.light}>
```

Reply:
also how come this was resolved?